tests/e2e-permissions.spec.ts

```diff
@@ -0,0 +1,375 @@
+        execSync(
+          `lxc auth identity-provider-group create ${idpGroupName}`,
+        ).toString(),
+      );
+    } catch (err) {
+      console.error("Error occurred:", err);
+    }
+  });
+
+  test.afterAll(() => {
+    try {
+      console.log(
+        "[Instance Deleted]",
+        execSync(`lxc delete ${instanceName} --force`).toString(),
+      );
+      console.log(
+        "[Profile Deleted]",
+        execSync(`lxc profile delete ${profileName}`).toString(),
+      );
+      console.log(
+        "[Network Deleted]",
+        execSync(`lxc network delete ${networkName}`).toString(),
+      );
+      console.log(
+        "[Storage Volume Deleted]",
+        execSync(
+          `lxc storage volume delete ${poolName} ${volumeName}`,
+        ).toString(),
+      );
+      console.log(
+        "[Custom ISO Volume Deleted]",
+        execSync(
+          `lxc storage volume delete ${poolName} ${customISOName}`,
+        ).toString(),
+      );
+      console.log(
+        "[Storage Pool Deleted]",
+        execSync(`lxc storage delete ${poolName}`).toString(),
+      );
+      console.log(
+        "[ACL Deleted]",
+        execSync(`lxc network acl delete ${aclName}`).toString(),
+      );
+      console.log(
+        "[Identity Untrusted]",
+        execSync(`lxc auth identity delete tls/${identityName}`).toString(),
+      );
+      console.log(
+        "[Group Deleted]",
+        execSync(`lxc auth group delete ${groupName}`).toString(),
+      );
+      console.log(
+        "[IDP Group Deleted]",
+        execSync(
+          `lxc auth identity-provider-group delete ${idpGroupName}`,
+        ).toString(),
+      );
+    } catch (err) {
+      console.error("Cleanup error:", err);
+    }
+  });
+
+  test("Cannot interact with instances", async ({ page }) => {
+    await gotoURL(page, "/ui/");
+    await page.getByRole("link", { name: "Instances", exact: true }).click();
+
+    await expect(
+      page.getByRole("button", { name: "Create instance" }),
+    ).toBeDisabled();
+    const title = await page
+      .getByRole("button", { name: "Create instance" })
+      .getAttribute("title");
+    expect(title).toContain("You do not have permission to create instances");
+
+    //Cannot start, restart, pause or stop instances
+    await visitInstance(page, instanceName);
+    await expect(
+      page.getByRole("button", { name: "Start", exact: true }),
+    ).toBeDisabled();
+    await expect(
+      page.getByRole("button", {
+        name: "You do not have permission to restart this instance",
+        exact: true,
+      }),
+    ).toBeDisabled();
+    await expect(
+      page.getByRole("button", {
+        name: "You do not have permission to freeze this instance",
+        exact: true,
+      }),
+    ).toBeDisabled();
+    await expect(
+      page.getByRole("button", {
+        name: "You do not have permission to stop this instance",
+        exact: true,
+      }),
+    ).toBeDisabled();
+  });
```

Comment:
I think instance actions have a bit more scope:
1. There are the migrate/create image/copy/export/delete actions in the header of the instance detail page
2. On the instance list there are the start/stop/etc actions and bulk actions 
3. On the instance list > side panel there are also the start/stop/etc actions.

I understand this is quite a scope increase. Would be fine with creating a future task to extend this later or doing it right now.